mscore/prefsdialog.ui

```diff
@@ -90,7 +90,7 @@
       <string>Preferences Tab Manager</string>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>3</number>
```

Comment:
This change shouldn't be in. Even if it might be harmless as it gets overwritten programatically